Cargo.lock

```diff
@@ -4414,6 +4424,16 @@ dependencies = [
  "utf-8 0.7.1 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
+name = "term"
+version = "0.5.2"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+dependencies = [
+ "byteorder 1.2.7 (registry+https://github.com/rust-lang/crates.io-index)",
+ "dirs 1.0.2 (registry+https://github.com/rust-lang/crates.io-index)",
+ "winapi 0.3.6 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
Hmm, I saw the discussion in https://github.com/servo/servo/issues/23430#issuecomment-494067148 and @atouchet said about it's good to remove `term` dependency from #23405. However, `color-backtrace` will introduce it again 😕 Not sure if it's fine to introduce this tool then 🤔

cc @paulrouget

Reply:
Yeah but this version of term already depends on winapi 0.3 so I guess that's ok